README.adoc

```diff
@@ -84,17 +89,18 @@ eval $(minikube docker-env)
 
 ==== Build the core riff components
 
-This builds the Function Sidecar, HTTP Gateway, Topic Controller, and Function Controller:
+This builds the Function Sidecar, HTTP Gateway, Topic Controller, Function Controller, and riff CLI:
 
 [source, bash]
 ----
 ./build-function-sidecar
 ./build-http-gateway
 ./build-topic-controller
 ./build-function-controller
+./build-riff-cli
 ----
 
-==== Build the function invokers
+==== Build the other function invokers
```

Comment:
Where are the build instructions for shell-function-invoker?

Reply:
included with core components list above